app/helpers/maintenance_tasks/tasks_helper.rb

```diff
@@ -149,26 +149,38 @@ def resolve_inclusion_value(task, parameter_name)
     # If the parameter has a `validates_inclusion_of` validator, return a dropdown list of options instead.
     def parameter_field(form_builder, parameter_name)
       inclusion_values = resolve_inclusion_value(form_builder.object, parameter_name)
-      return form_builder.select(parameter_name, inclusion_values, prompt: "Select a value") if inclusion_values
+      return form_builder.select(
+        parameter_name,
+        inclusion_values,
+        {
+          prompt: "Select a value",
+        },
+      ) if inclusion_values
 
       case form_builder.object.class.attribute_types[parameter_name]
       when ActiveModel::Type::Integer
-        form_builder.number_field(parameter_name)
+        form_builder.number_field(parameter_name, class: "input")
       when ActiveModel::Type::Decimal, ActiveModel::Type::Float
-        form_builder.number_field(parameter_name, { step: "any" })
+        form_builder.number_field(parameter_name, { step: "any", class: "input" })
       when ActiveModel::Type::DateTime
-        form_builder.datetime_field(parameter_name) + datetime_field_help_text
+        form_builder.datetime_field(parameter_name, class: "input") + datetime_field_help_text
       when ActiveModel::Type::Date
-        form_builder.date_field(parameter_name)
+        form_builder.date_field(parameter_name, class: "input")
       when ActiveModel::Type::Time
-        form_builder.time_field(parameter_name)
+        form_builder.time_field(parameter_name, class: "input")
       when ActiveModel::Type::Boolean
-        form_builder.check_box(parameter_name)
+        form_builder.check_box(parameter_name, class: "checkbox")
       else
         form_builder.text_area(parameter_name, class: "textarea")
       end
     end
 
+    # Return whether the parameter should be rendered as a dropdown list.
+    def select?(form_builder, parameter_name)
+      inclusion_values = resolve_inclusion_value(form_builder.object, parameter_name)
+      inclusion_values.is_a?(Array)
```

Comment:
nit but I think it's going to be even more readable (and shorter) if inlined
```suggestion
      resolve_inclusion_value(form_builder.object, parameter_name).is_a?(Array)
```